### ISSUE TITLE
Can't see Jade info anymore. 

### ISSUE THREAD

**Issue by CephalonAxii** [Snownee/Jade#178]
### Mod loader

Forge

### Minecraft version

1.16.5

### Mod version

jade-1.16.4-2.8.3

### Modloader version

1.16.5-forge-36.2.35

### Modpack info

Majority of Medieval Modpack quite a lot more mods added: https://www.curseforge.com/minecraft/modpacks/medieval-mc-modpack/relations/dependencies?page=5 

### If bug:

- [ ] Can you reproduce this issue with relevant mods only?

### Issue description

I was able to see block and mob descriptions from Jade until recently. I figure that there is either a config change that I did and didn't realize what, or I downloaded a mod that somehow conflicts with Jade. But I removed and readded all the recent mods I did and it was still not showing. I also tried seeing the config settings again but I don't really know what would cause it to disappear.

**Comment by Snownee** [comment]
did you press Numpad 1?

**Comment by CephalonAxii** [comment]
I don't think so. But I just reloaded the game and pressed it just now. No
change.

**Comment by CephalonAxii** [comment]
Okay, it was a bit of a riggamarole, but I figured out to fix it. I deleted Jade from my mods, and the configurations from minecraft then redownloaded a slightly older version of Jade and replaced my Forge Installer with the latest version. Then I ran the game with only Jade and JEI and it worked. Now I have all my mods back and everything working great. Don't think all of the above were required to fix it, but redundancey is safety XD. Thank you for you time in trying to help me. I really appreciate it.
